server/controllers/postController.ts

```diff
@@ -12,7 +12,7 @@ async function postPosts(req, res, next) {
 
 async function getPosts(req, res, next) {
 	if ('number' in req.query && 'page' in req.query) {
-		const result = await postManager.getPostsPage(req.query.number, req.query.page);
+		const result = await postManager.getPostsPage(req.query.number, req.query.page, req.query.sort);
```

Comment:
It's not critical, but I'd suggest to use in these cases destruction of even rest parameters in some cases.